anda/system/si-cik-amdgpu/si-cik-amdgpu.spec

```diff
@@ -0,0 +1,46 @@
+Name:          si-cik-amdgpu
+Version:       1
```

Comment:
can you use the commit instead? also change Source0

also Requires(post): dracut and postun